cloudhub-2/modules/ROOT/pages/ch2-ha-dr.adoc

```diff
@@ -0,0 +1,269 @@
+
+== High Availability Versus Disaster Recovery
+
+High availability (HA) is the measure of a system’s ability to remain accessible despite a system component failure. You generally implement HA by building in multiple levels of fault tolerance or load balancing capabilities into a system. In CloudHub 2.0, you can achieve high availability by distributing multiple replicas across three Availability Zones (AZs) without requiring additional configuration.
+
+Disaster recovery (DR) refers to the process of restoring a system to an acceptable previous state after a natural or man-made disaster, such as flooding, tornadoes, earthquakes, fires, power failures, server failures, and misconfigurations.
+
+While they both increase overall availability, the difference is that with HA there's generally no loss of service. HA retains the service, and DR retains the data, but with DR, there's usually a slight loss of service while the DR plan executes and the system restores.
+
+=== Key Definitions
+
+These key terms are essential to understanding HA and DR strategies in CloudHub 2.0:
+
+Recovery Time Objective (RTO):: This is the maximum amount of downtime a business can tolerate. The RTO represents the time it takes for the system to recover after a business disruption.
+
+Recovery Point Objective (RPO):: This is the maximum amount of time acceptable for data loss after a disaster. The RPO influences the frequency of data backups.
+
+== High Availability in CloudHub 2.0
+
+CloudHub 2.0 implements high availability through several built-in mechanisms, which MuleSoft manages automatically, and require no additional configuration.
+
+=== Redundant Locations
+
+MuleSoft Anypoint works with instances from one place. This mechanism copies transactions in near real time across two or more availability zones in completely different, separate locations. This ensures service continuity if one location fails.
+
+=== Regular Switching
+
+MuleSoft regularly switches among locations for maintenance, compliance, and disaster recovery purposes. This keeps all locations up-to-date and able to handle the load if required.
+
+=== Ingress and Egress Traffic Management
+
+CloudHub 2.0 uses a sophisticated ingress and egress traffic management system that routes traffic optimally, even during maintenance or unexpected downtimes.
+
+=== VPN Tunnel Updates
+
+During VPN tunnel updates, the system sets a lower outbound multiexit discriminator (MED) value on the other tunnel. This maintains connectivity.
+
+=== Out-of-the-Box (OOTB) High Availability
+
+High availability is possible out-of-the-box by configuring multiple replicas across three availability zones (AZs).
```

Comment:
Earlier (line 32), you say "no additional configuration". 
Also, doesn't OOTB basically mean "works without configuration"?